src/games/strategy/triplea/settings/IntegerValueRange.java

```diff
@@ -0,0 +1,15 @@
+package games.strategy.triplea.settings;
+
+public class IntegerValueRange {
+
+  public final int lowerValue;
+  public final int upperValue;
+  public final int defaultValue;
```

Comment:
As long as they are immutable, this is slightly better compared to having getters because it less code. Getters provide no additional benefit in this case.